operators/tokenizer/sentencepiece_tokenizer.cc

```diff
@@ -53,6 +54,16 @@ void KernelSentencepieceTokenizer::Compute(const ortc::Tensor<std::string>& inpu
       if (add_bos) {
         content.push_back(tokenizer_.bos_id());
       }
+    } else if (xlm_roberta.has_value() && xlm_roberta) {
+      // As per HF, the first "real" token "," has position 4 in the XLMRobertaTokenizer vocab
```

Comment:
It's good to paste the XLM-Roberta tokenizer code snippet here to help understanding.

Reply:
done!